custom/modules/Users/SticImpersonate/ImpersonateUtils.js

```diff
@@ -0,0 +1,108 @@
+        return;
+    }
+    showImpersonationIndicator();
+    showImpersonationMessage();
+});
+
+/**
+ * Shows a message at the top of the page content indicating that the user is impersonating another user. 
+ */
+function showImpersonationMessage() {
+    document.querySelectorAll('#stic-impersonation-message').forEach(el => el.remove());
+
+    var ref = document.getElementById('pagecontent');
+    if (ref.length === 0) {
+        console.warn('Did not find #pagecontent, cannot insert impersonation message');
+        return;
+    }
+    var message = document.createElement('div');
+    message.id = 'stic-impersonation-message';
+    message.className = 'msg-fatal-lock';   
+    message.innerHTML = `
+        ${modImpersonate.LBL_IMPERSONATE_ALERT_TITLE} 
+        ${modImpersonate.LBL_IMPERSONATE_ALERT_DESCRIPTION}
+        <strong>${impersonationData.target_user_name}</strong>
+        ${modImpersonate.LBL_IMPERSONATE_ALERT_STOP_DESCRIPTION}
+    `;
+    // Inserting as the first child of the reference node
+    ref.insertBefore(message, ref.firstChild);
+}
+
+/**
+ * Shows an impersonation indicator in the top navigation bar.
+ */
+function showImpersonationIndicator() {
+    // Removing previous indicators
+    document.querySelectorAll('#stic-impersonation-indicator').forEach(el => el.remove());
+
+    // Searching all #globalLinks to insert after each one desktop/mobile/tablet
+    var refs = document.querySelectorAll('#globalLinks');
+    if (refs.length === 0) {
+        console.warn('Did not find #globalLinks, cannot insert impersonation message');
+        return;
+    }
+
+    refs.forEach(ref => {
+        var indicator = document.createElement(ref.tagName.toLowerCase());
+        indicator.id = 'stic-impersonation-indicator';
+        indicator.className = 'dropdown nav navbar-nav desktop_notifications impersonate-indicator';
+
+        indicator.innerHTML = `
+            <button class="alertsButton btn dropdown-toggle suitepicon suitepicon-action-view impersonate-indicator-icon user-menu-button"  data-toggle="dropdown" aria-expanded="false">
+            </button>
+            <ul class="dropdown-menu user-dropdown impersonation-dropdown" role="menu" aria-labelledby="with-label">
+                <li role="presentation" class="dropdown-header">
+                    ${modImpersonate.LBL_IMPERSONATE_ALERT_TITLE}
+                </li>
+                <li role="presentation">
+                    <a href="javascript:void(0)">
+                        <strong>${modImpersonate.LBL_IMPERSONATE_TARGET_USER}:</strong> ${impersonationData.target_user_full_name}
+                    </a>
+                </li>
+                <li role="presentation">
+                    <a href="javascript:void(0)">
+                        <strong>${modImpersonate.LBL_IMPERSONATE_ORIGINAL_USER}:</strong> ${impersonationData.original_user_full_name}
+                    </a>
+                </li>
```

Comment:
```suggestion
                <li role="presentation">
                        <strong>${modImpersonate.LBL_IMPERSONATE_TARGET_USER}:</strong> ${impersonationData.target_user_full_name}
                </li>
                <li role="presentation">
                        <strong>${modImpersonate.LBL_IMPERSONATE_ORIGINAL_USER}:</strong> ${impersonationData.original_user_full_name}
                </li>
```
Creo que la información del usuario suplantado y del usuario original no necesita ser un enlace, ya que no realiza ninguna acción ni redirección.